pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/admin/UserAdminTestUtil.java

```diff
@@ -407,4 +408,13 @@ private static void deleteInstance(int instanceId) {
         });
     }
 
+    public void setUpByConfig(String ddpInstanceId, String ddpGroupId) {
+        if (StringUtils.isBlank(ddpInstanceId) || StringUtils.isBlank(ddpGroupId)) {
```

Comment:
I need the study roles to be set up by the instance id and group id from my previous set up of juniper